test/unit/libraries/Tiebreaker.t.sol

```diff
@@ -0,0 +1,193 @@
+        assertFalse(context.sealableWithdrawalBlockers.contains(address(mockSealable1)));
+    }
+
+    function test_SetTiebreakerCommittee() external {
+        address newCommittee = address(0x123);
+
+        vm.expectEmit();
+        emit Tiebreaker.TiebreakerCommitteeSet(newCommittee);
+        Tiebreaker.setTiebreakerCommittee(context, newCommittee);
+
+        assertEq(context.tiebreakerCommittee, newCommittee);
+    }
+
+    function test_SetTiebreakerCommittee_WithExistingCommitteeAddress() external {
+        address newCommittee = address(0x123);
+
+        Tiebreaker.setTiebreakerCommittee(context, newCommittee);
+        Tiebreaker.setTiebreakerCommittee(context, newCommittee);
+    }
+
+    function test_SetTiebreakerCommittee_RevertOn_ZeroAddress() external {
+        vm.expectRevert(abi.encodeWithSelector(Tiebreaker.InvalidTiebreakerCommittee.selector, address(0)));
+        Tiebreaker.setTiebreakerCommittee(context, address(0));
+    }
+
+    function testFuzz_SetTiebreakerActivationTimeout(uint32 minTimeout, uint32 maxTimeout, uint32 timeout) external {
+        vm.assume(minTimeout < timeout && timeout < maxTimeout);
+
+        Duration min = Duration.wrap(minTimeout);
+        Duration max = Duration.wrap(maxTimeout);
+        Duration newTimeout = Duration.wrap(timeout);
+
+        vm.expectEmit();
+        emit Tiebreaker.TiebreakerActivationTimeoutSet(newTimeout);
+
+        Tiebreaker.setTiebreakerActivationTimeout(context, min, newTimeout, max);
+        assertEq(context.tiebreakerActivationTimeout, newTimeout);
+    }
+
+    function test_SetTiebreakerActivationTimeout_RevertOn_InvalidTimeout() external {
+        Duration minTimeout = Duration.wrap(1 days);
+        Duration maxTimeout = Duration.wrap(10 days);
+        Duration newTimeout = Duration.wrap(15 days);
+
+        vm.expectRevert(abi.encodeWithSelector(Tiebreaker.InvalidTiebreakerActivationTimeout.selector, newTimeout));
+        Tiebreaker.setTiebreakerActivationTimeout(context, minTimeout, newTimeout, maxTimeout);
+
+        newTimeout = Duration.wrap(0 days);
+
+        vm.expectRevert(abi.encodeWithSelector(Tiebreaker.InvalidTiebreakerActivationTimeout.selector, newTimeout));
+        Tiebreaker.setTiebreakerActivationTimeout(context, minTimeout, newTimeout, maxTimeout);
+    }
+
+    function test_IsSomeSealableWithdrawalBlockerPaused() external {
+        mockSealable1.pauseFor(1 days);
+        Tiebreaker.addSealableWithdrawalBlocker(context, address(mockSealable1), 2);
+
+        bool result = Tiebreaker.isSomeSealableWithdrawalBlockerPaused(context);
+        assertTrue(result);
+
+        mockSealable1.resume();
+
+        result = Tiebreaker.isSomeSealableWithdrawalBlockerPaused(context);
+        assertFalse(result);
+
+        mockSealable1.setShouldRevertIsPaused(true);
+
+        result = Tiebreaker.isSomeSealableWithdrawalBlockerPaused(context);
+        assertTrue(result);
+    }
+
+    function test_CheckTie() external {
```

Comment:
It's preferred to use `_HappyPath_[SomeConcreteScenario]` suffix for the happy path unit tests: `test_checkTie_HappyPath()`